app/assets/javascripts/messages.coffee

```diff
@@ -0,0 +1,25 @@
+# jQuery(document).on 'turbolinks:load', ->
```

Comment:
Is this whole file commented out? Need to include it at all?

Reply:
yep I will omit this file, good point.